bundles/binding/org.openhab.binding.zwave/database/products.xml

```diff
@@ -1705,6 +1705,15 @@
             <Label lang="en">Roller Shutter</Label>
             <ConfigFile>fibaro/fgrm222.xml</ConfigFile>
         </Product>
+		<Product>
```

Comment:
Sorry - minor issue - can you just sort out the indentation so it's inline with the other part of the file.